src/Screens/Phpunit.php

```diff
@@ -2,16 +2,21 @@
 
 namespace Spatie\PhpUnitWatcher\Screens;
 
+use Spatie\PhpUnitWatcher\Notifier;
 use Symfony\Component\Process\Process;
 
 class Phpunit extends Screen
 {
     /** @var string */
     protected $phpunitArguments;
 
-    public function __construct(string $phpunitArguments = '')
+    /** @var bool */
+    protected $notifications;
```

Comment:
This variables does not hold the notifications itself, only if a flag if if they should be used.

Renaming this to `sendNotifications` makes this more clear.